src/container/styles.scss

```diff
@@ -32,7 +32,9 @@
 
 .header {
   background-color: awsui.$color-background-container-header;
-  border-radius: awsui.$border-radius-container;
+  border-top-left-radius: awsui.$border-radius-container;
+  border-top-right-radius: awsui.$border-radius-container;
```

Comment:
What happens to the bottom border radius in a non cards variant?

Reply:
Is it also expected that lines 80 and 81 can override this?

Reply:
1. in non-cards variant the content is always displayed, even if empty.
2. yes, when the container is stuck the borders at the top are removed.